admin/acceptance_tests/restriction_areas_test.py

```diff
@@ -0,0 +1,136 @@
+# pylint: disable=no-self-use
+
+
+import json
+import pytest
+import re
+from geoalchemy2.shape import from_shape
+from shapely.geometry import box, Polygon, shape
+
+from . import AbstractViewsTests
+
+
+@pytest.fixture(scope='class')
+@pytest.mark.usefixtures('dbsession')
+def restriction_area_test_data(dbsession):
```

Comment:
You can already set scope to 'function' and add 'transact' in usefixture.
Note that you need to add trasact also as parameter and del transact for linter.
In that case you should remove internal transaction.
I would prefer having a flush before return.